### DISCUSSION TITLE
Manning book on Fluent Bit

### DISCUSSION THREAD

**Discussion by mp3monster** [fluent/fluent-bit#8162] [Show and tell]
Manning has just released a book on Fluent Bit into their Early Access Program (MEAP) - you can find the details at [Fluent Bit with Kubernetes](https://www.manning.com/books/fluent-bit-with-kubernetes?utm_source=Phil&utm_medium=affiliate).

[Calyptia ](https://calyptia.com/)has sponsored the book

**Comment by patrick-stephens** [comment]
There's a free option here as well: https://calyptia.com/fluent-bit-with-kubernetes-manning

**Comment by mp3monster** [comment]
The MEAP edition of the book now has all 10 chapters available

**Comment by mp3monster** [comment]
The book has had a title change to Logs and Telemetry with Fluent Bit - but the content remains the same.  I'm also excited to say that the title has been sent to the printers and the various ebook channels should be updated in the next couple of weeks including Amazon, O'Reilly (Safari) as well as Manning.

Additional content can be found at https://blog.mp3monster.org/category/technology/fluent-bit/ and from Manning https://www.manning.com/books/logs-and-telemetry?a_aid=Phil&  (free version sponsored by Calyptia is out there as well - and can expect to be updated)
